### PR TITLE
add center variants to money rich text block

### PR DESCRIPTION
**Pull request by Segger87**
image

### CODE REVIEW

File: src/themes/money/theme.json [hunk 1]
```diff
@@ -1304,6 +1304,18 @@
           },
           "boxShadow": "0px 2px 3px rgba(24, 24, 61, 0.06), 0px 5px 5px rgba(24, 24, 61, 0.06), 0px 8px 7px rgba(24, 24, 61, 0.06)",
           "borderRadius": "md"
+        },
+        "panel-primary-text-align-center": {
+          "variant": "elements.rich-text-block.variants.panel-primary",
+          "textAlign": ["center", "center"],
```
Comment (meghaniankov) [inline src/themes/money/theme.json:1328]: does it need to be split between mobile/desktop if both are center?

**Comment by jonathan-fielding** [comment]
@Segger87 you should check the spacing with rob, looks bit wierd

File: src/themes/money/theme.json (hunk 1 above)
Comment (Segger87) [reply src/themes/money/theme.json:1328]: ive already updated this, it was copied from Uswitch theme.

**Comment by Segger87** [comment]
> @Segger87 you should check the spacing with rob, looks bit wierd

the ticket is to fix centering on rich text blocks, if there is additional spacing issues with the panel thats a seperate task.

**Review by barneyfoxuk — APPROVED** [review]

**Pull request merged**